web/src/views/repo/settings/Actions.vue

```diff
@@ -1,109 +0,0 @@
-<template>
```

Comment:
Would put this change in a separate PR.

Reply:
Just leave it without an icon then. I am fine with removing the tab in general, found the save button to be in the middle of the page a bit wrong on the other hand.

Reply:
Fair. Ok as this might need discussion I'll add an icon for now so we can merge this PR in a consistent state.